spec/mailers/candidate_mailer_spec.rb

```diff
@@ -364,6 +364,17 @@
       'pending condition text' => 'You still need to meet the following condition',
       'pending condition' => 'Be cool',
     )
+
+    context 'when new reference flow is active' do
+      before do
+        FeatureFlag.activate(:new_references_flow)
+        application_form.recruitment_cycle_year = 2023
```

Comment:
similarly, could this be shifted into the declaration of `application_form` and the magic number replaced?